views/comment.py

```diff
@@ -64,13 +87,14 @@ def add_comment(data):
 
         db_cursor.execute(
             """
-                INSERT INTO Comments (post_id, author_id, content)
-                VALUES (?,?,?)
+                INSERT INTO Comments (post_id, author_id, content, date)
+                VALUES (?,?,?,?)
             """,
             (
                 data["post_id"],
                 data["author_id"],
                 data["content"],
+                data["date"],
```

Comment:
Instead of making the client provide a date by using `data["date"]`, you can instead use `datetime.now()`. In this way, the client will still only need to provide the first three items, and won't need to worry about the date. This is how they showed us to do it in the code that was provided for us in the `user` creation.

Reply:
the client still doesnt need to provide the date. If you look at the NewComments page I have that build directly into the fuction.